website/ipam.json

```
@@ -368,10 +368,18 @@
     {
       "address": "44.34.128.222",
       "name": "k4sof",
-      "site": "client",
-      "asset": null,
+      "site": "mcn",
+      "asset": "wlan1-gateway",
```

Keep asset null unless you have a corresponding entry on the memhamwan/inventory repo, and in that case you’d put the issue ID here.

```suggestion
      "asset":null,
```

---

How do we want to account for devices with multiple IPs on different interfaces? It doesn't seem like we have a the ability to address that (and I was cheating using that field).

---

The thing that this is issuing is an “interface” field — asset already tracks device. So for instance in your case, you could add an asset to the repo and then trace your different addresses back to that if you’d like.

This whole list is quick-and-dirty for now. I don’t like this as a long term solution and think that we need to migrate to a real IPAM. But if we can get this data right for now, it’ll make it simple for us to migrate it into an IPAM once we have that in place.